curvefs/src/metaserver/s3/metaserver_s3.cpp

```diff
@@ -53,5 +53,21 @@ int S3ClientImpl::Delete(const std::string& name) {
 
     return ret;
 }
+
+int S3ClientImpl::DeleteBatch(const std::list<std::string>& nameList) {
+    std::list<Aws::String> keyList;
```

Comment:
Is there a limit to the size of the keylist？

Reply:
key list has no size, but key list will cut to batch size, use batch size to delete from s3.
for example, namelist has 1000 key to delete, batch size is 100, so it will send delete request to s3 every 100 key，totally call delete 10 times.

Reply:
the batch size is config in metaserver.conf , default is 100
s3.batchsize=100